### DISCUSSION TITLE
Potential copper investigation

### DISCUSSION THREAD

**Discussion by emoon** [dirkwhoffmann/vAmiga#462] [General]
https://github.com/tonioni/WinUAE/issues/188 I haven't looked at how it behaves in vAmiga, but might be worth taking a look

**Comment by dirkwhoffmann** [comment]
vAmiga has the same bug:

<img width="988" alt="Bildschirmfoto 2021-01-19 um 20 35 16" src="https://user-images.githubusercontent.com/12561945/105083989-032ff180-5a96-11eb-9ea9-d5465d036391.png">

<img width="988" alt="Bildschirmfoto 2021-01-19 um 20 35 21" src="https://user-images.githubusercontent.com/12561945/105084007-088d3c00-5a96-11eb-94f4-76e467bc4d2a.png">

**Comment by emoon** [comment]
I see. Likely worth investigating then :)
